libopae/CMakeLists.txt

```diff
@@ -106,10 +106,20 @@ if(CMAKE_BUILD_TYPE STREQUAL "Release")
   set_property(TARGET opae-c PROPERTY C_VISIBILITY_PRESET hidden)
 endif(CMAKE_BUILD_TYPE STREQUAL "Release")
 
-# Binary install locations
-install(TARGETS opae-c
-  LIBRARY DESTINATION lib
-  COMPONENT opaeclib)
+#install(TARGETS opae-c
+#  LIBRARY DESTINATION ${LIB_INSTALL_DIR}
```

Comment:
Why is this commented out? Can it be refactored to use a CACHE variable so we don't have to call "get_property" in line 113?

Reply:
Yes that can be done. I will store it in a cached variable and use that variable throughout.